core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -254,11 +267,18 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
 
   def poll(): Unit = {
     val wakeupTask = context.system.scheduler.scheduleOnce(settings.wakeupTimeout) {
+      log.warning("KafkaConsumer poll is taking significantly longer ({}ms) to return from poll then the configured poll interval ({}ms). Waking up consumer to avoid thread starvation. Thread dump:", settings.wakeupTimeout.toMillis, pollTimeout().toMillis)
```

Comment:
remove the "Thread dumps:" at the end, belongs to next log message

Reply:
Doh